tutor/src/screens/scores-report/styles/table.scss

```diff
@@ -346,41 +342,38 @@ $pie-color-due: #268FBE;
   }
   .popover-content {
     padding: 3px;
+
     .late-status {
-      height: 60px;
+      display: flex;
+      flex-direction: row;
+
       .description {
-        height: inherit;
-        display: table;
-        width: 48%;
-        float: left;
         font-weight: bold;
-      }
+        display: flex;
+        flex-direction: row;
+        align-items: center;
 
+        .title {
+          white-space: nowrap;
+          margin: 0 5px 0 10px;
+        }
+
+        .status {
+          margin: 0 10px 0 5px;
+          color: $tutor-primary;
+        }
+      }
       .body {
         margin: 0 10px;
       }
-
-      .title {
-        display: table-cell;
-        vertical-align: middle;
-        text-align: center;
-
-      }
-      .status {
-        display: table-cell;
-        vertical-align: middle;
-        color: $tutor-primary;
-        text-align: center;
+      .late-button {
+        flex: 1;
+        margin: 3px;
+        white-space: normal;
+        text-transform: none !important;
+        font-size: 1.1em !important;
```

Comment:
yay for replacing `table-cell` with `flex`!   

Why is the `!important` needed?  I'm guessing it's to override of some other rule?  If it doesn't make it too convoluted I'd prefer to match the specific rule by adding more specific selectors.

If that does make it too complex can we add a comment indicating what it's overriding in hopes (which yeah, probably won't happen) that we can remove the !important if it's ever no longer needed?

Reply:
i'm not sure i didn't modify these styles i just moved them, do you want me to try to clean it up some more?

Reply:
oh, I see.  Yes they had the `!important` originally.  Ok, you've def made the styles better so I think we can cut off the improvements here otherwise we could go on forever 😁